src/BaywatchOperation.php

```diff
@@ -4,11 +4,34 @@
 
 use Drupal\Core\Serialization\Yaml;
 use Drupal\Core\File\FileSystemInterface;
+use Drupal\user\Entity\Role;
 
 
 class BaywatchOperation
 {
 
+  /**
+   * Helper to install a module.
+   *
+   * @param string $module
+   *   Module name.
+   *
+   * @throws \Exception
+   *   When module already installed.
+   */
+  function baywatch_install_module($module) {
+    /** @var \Drupal\Core\Extension\ModuleHandler $moduleHandler */
+    $moduleExists = \Drupal::service('module_handler')->moduleExists($module);
+    // Check if module is both installed and enabled.
+    if (!$moduleExists) {
+      // If not, install the queue_mail module.
+      \Drupal::service('module_installer')->install([$module]);
+    }
+    if ($moduleExists) {
+      throw new \Exception(sprintf('Unable to install module %s', $module));
```

Comment:
Should this fail quietly instead of throwing an exception?

Reply:
I added that cause at least it will show up in the logs saying the module is not installed, in case someone is looking into the build logs.